crates/librqbit/README.md

```diff
@@ -1 +1,128 @@
-../README.md
\ No newline at end of file
+# librqbit
+
+A torrent library 100% written in rust
+
+## Basic example
+This is a simple on how to use this library  
+This program will just download a simple torrent file with a Magnet link
+
+```rust
+use std::error::Error;
+use std::path::PathBuf;
+use std::sync::Arc;
+use std::time::Duration;
+use librqbit::{http_api_client, Magnet};
+use librqbit::session::{AddTorrentResponse, ListOnlyResponse, ManagedTorrentState, Session};
+use librqbit::spawn_utils::BlockingSpawner;
+use size_format::SizeFormatterBinary as SF;
+use tokio::spawn;
+
+const MAGNET_LINK: &str = "magnet:?..."; // Put your magnet link here
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn Error>>{
+
+    let spawner = BlockingSpawner::new(false);
+
+    // This function will print the torrent properties every 1s

```

Comment:
this block contributes to example size a lot, but isn't required for the example to work (to download the torrent).
I'd remove it to showcase the simplicity of use